pipeline/train/configs/model/student.base.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/browsermt/students/tree/master/train-student/models/student.base
+# the difference with the "tiny" configuration is the dimensionality of transformer-dim-ffn and dim-emb
```

Comment:
Suggestion: Is there a way to link here in the code (or in the docs) to your results? This could be a github PR or issue or w&b link. This will help future contributors to have context as to why things exist the way they do.

Reply:
It's a good idea to add a link to the experiment I did but there are also other ones that show the difference between base and tiny, for example, hplt docs. The experiment I did in fact doesn't matter here.  We're just reproducing the Bergamot recipe which provided two student configurations. We supported only one of them before. I guess I'll add a link to the issue if it helps to understand the context better.